sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -1444,10 +1444,6 @@ export class ApplicationService extends RecordDataModelService<Application> {
         student: {
           id: studentId,
         },
-        studentScholasticStandings: {
```

Comment:
why was these removed, same in line 1507

Reply:
The condition was introduced on the previous PR for the same ticket and was preventing a record without a scholastic standing from being returned.